protractor.conf.js

```diff
@@ -10,6 +10,10 @@ var q = require('q');
 
 var config = {
   
+  seleniumPort:      4444,
```

Comment:
If you are using a local protractor, you shouldn't need to specify these properties.